typings/types.d.ts

```diff
@@ -125,3 +125,14 @@ export interface IWarning {
 export interface IWorkspaceSettings {
     supportedRecipeTypes: string;
 }
+
+export interface IFactory {
```

Comment:
hello, should it use as well the DTO types created by Che / TypeScript generator ?
cc @evidolob

Reply:
Yes, it should use new DTO types